frontend/app/lib/sprites/tyton.js

```diff
@@ -11,6 +11,9 @@ class Tyton extends Phaser.Sprite {
           "tyton");
     this.width = 120;
     this.height = 120;
+    this.health = 100;
+    this.power = 0;
+    this.damage = { 'default': 10 };
```

Comment:
take this out. the concept of `damage` is a property of the thing that did damage, not a property of the Tyton. if we are thinking of `toughness` as a property of Tyton, that's a useful concept but don't worry about `toughness` right now because we don't need it yet